src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -41,6 +44,13 @@
 	private transient String priorityParameterName;
 	private List<LockableResource> resources;
 
+
+	/**
+	 * Only used when this lockable resource is tried to be locked by {@link LockStep},
+	 * otherwise (freestyle builds) regular Jenkins queue is used.
+	 */
+	private List<QueuedContextStruct> queuedContexts = new ArrayList<QueuedContextStruct>();
```

Comment:
Moving this field here makes it hidden to static serialized state (needed for restart-ability).